### PR TITLE
fix: add an optional canister_settings parameter to the create_canister function

### PR DESCRIPTION
**Pull request by saikatdas0790**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -4,6 +4,7 @@ use ciborium::de::from_reader;
 use ic_cdk::api::management_canister::main::{
     CanisterId, CanisterIdRecord, CanisterInstallMode, CreateCanisterArgument, InstallCodeArgument,
 };
+use ic_cdk::api::management_canister::provisional::CanisterSettings;
```
Comment (frederikrothenberger) [inline src/lib.rs:7]: You should rather use `ic_cdk::api::management_canister::main::CanisterSettings` and add it to the list above. The `provisional` module is just a re-export.

**Comment by frederikrothenberger** [comment]
Thanks for the PR, I left a small comment. When fixed, I'll merge and make a new release for you.

File: src/lib.rs (hunk 1 above)
Comment (saikatdas0790) [reply src/lib.rs]: Done

**Review by frederikrothenberger — APPROVED** [review]
Thanks a lot! 🙂

**Pull request merged**